docs/examples/dodge-the-burgers.md

```diff
@@ -0,0 +1,59 @@
+# Dodge the Burgers
+
+Dodge the burgers and eat the fruits!
+
+```blocks
+enum SpriteKind {
+    Player,
+    Projectile,
+    Food,
+    Enemy
+}
+let projectile: Sprite = null
+let otherSprite: Sprite = null
```

Comment:
should remove the otherSprite and sprite declarations / initializations so it doesn't generate nulls in blocks

Reply:
Yup, need to figure out who adds those.